src/styles/features/pathways/network.css

```diff
@@ -19,7 +19,7 @@
 }
 
 .network-sidebar-open {
-  width: calc(100vw - var(--view-sidebar-width));
+  width: calc(100vw - var(--view-sidebar-width)) !important;
```

Comment:
I don't think this is necessary.  Can you confirm if it works with only the above js change?